packages/core/useConfirmDialog/index.md

```diff
@@ -0,0 +1,238 @@
+---
+category: Utilities
+---
+
+# useConfirmDialog
+
+Creates event hooks to support modals and confirmation dialog chains. It returns 3 functions and 3 hooks.
```

Comment:
```suggestion
Creates event hooks to support modals and confirmation dialog chains.
```

IMO this info is unnecessary / can be left out.
Assuming someone would add another function to this feature, they will miss this numbers to update